src/main/java/duke/Date.java

```diff
@@ -0,0 +1,105 @@
+package duke;
+
+import java.util.Hashtable;
+import java.util.regex.Pattern;
+
+/**
+ * Represents a date and time of an event. A <code>Date</code> object corresponds
+ * to a date and time represented as a string e.g. "01/01/2000 0000"
+ */
```

Comment:
Innovative method! Did you consider using the in-built LocalDateTime class, however?